tensor.go

```diff
@@ -79,8 +79,11 @@ func (a Tensor) Print() {
 }
 
 // Close the tensor
-func (a Tensor) Close() {
-	C.Tensor_Close(*a.T)
+func (a *Tensor) Close() {
```

Comment:
Because we are going to mutate a.T, so we want a being a pointer instead of a copy of the real a.